src/V3Number.cpp

```diff
@@ -2037,11 +2044,11 @@ V3Number& V3Number::opModDivGuts(const V3Number& lhs, const V3Number& rhs, bool
         }
         t = un[j + vw] - k;
         un[j + vw] = t;
-        this->m_value[j].m_value = qhat;  // Save quotient digit
+        this->m_data.num()[j].m_value = qhat;  // Save quotient digit
```

Comment:
```suggestion
        m_data.num()[j].m_value = qhat;  // Save quotient digit
```

Not sure why the this->'s were there, perhaps historical but please clean them up since in there, thanks.

Reply:
I've fixed all occurences in the file.